docs/projects/P4/index.md

```diff
@@ -10,34 +10,37 @@
+Once your CEO has finally calmed down, your manager calls an all-hands meeting. At the meeting, everyone agrees that you don’t have the resources to build a completely new Q&A forum and also automate the whole question-answering problem.
 
-However, to better compete with Piazza, your CEO and manager decide to put your team in charge of exploring using an LLM to implement a translation feature. This feature will automatically translate posts written in languages other than English into English. This initiative aims to foster inclusivity and ensure that language barriers do not hinder participation in the forum.
+But, to better compete with Piazza, your CEO and manager decide to put your team in charge of exploring using an LLM to implement a translation feature. This feature will automatically translate posts written in languages other than English into English. This initiative aims to foster inclusivity and ensure that language barriers do not hinder participation in the forum.
 
-Your manager wants you to:
+In light of this new direction, your manager wants you to:
 
 1. **Architect the feature** - reason about, document and select an appropriate architecture for integrating this feature into the existing system.
 2. **Build an experimental integration** with the existing NodeBB codebase to evaluate feasibility.
 3. **Evaluate the feature** and decide whether the feature should be fully implemented.
 
+As everything is just settling down, your CEO bursts back into the room!! There's a new catch: the company is strapped for funding and must be very careful with its spending. As a result, your team is now required to conduct all experimentation and implementation of the LLM-powered translation feature **using only the Azure credits currently at your disposal**. If your team successfully stays within your available credits and doesn't need to request any more, you’ll earn **bonus points** for your ingenuity and efficiency.
+
+Some of you may think that it’s unfair for this change to come up unexpectedly, especially since we didn’t inform you before P3, but keep in mind that this is just like real life! Working in a company means having to deal with unexpected circumstances, which more often than not contain some aspect concerning money. We're looking to see how well you can adapt to this new situation, and we want you to have this experience before you're dealing with the pressure of real company money. Rest assured, if your team needs more credits to finish the project, we will not be taking off points.
+
 ## Deliverables and Deadlines
 There are three (3) deadlines for the project. This project is worth a total of 125 points.
 
-**Checkpoint #1 Deliverables** – 55 pts - due Thursday, March 28, 11:59pm
+**Checkpoint #1 Deliverables** – 55 pts - due Thursday, November 7, 11:59pm
```

Comment:
Update the due dates to be 

Checkpoint #1 due by Friday, Nov 8th and the integration due by Thursday, Nov 14th

as described from Slack